rb/lib/selenium/webdriver/edge/legacy_support.rb

```diff
@@ -27,8 +27,7 @@ module LegacySupport
            send_keys_to_active_element get_window_handles get_current_window_handle
```

Comment:
From what I can see, all the methods in `legacy_support.rb` are overridden in the `Edge::Bridge` already. So I think if everything is pointed at `Remote::Bridge`, then it should be ok to delete.